gui/build.py

```diff
@@ -60,7 +60,11 @@ def do_run(options):
     do_uic(options)
     do_rcc(options)
     env = dict(os.environ)
-    env['PYTHONPATH'] = '.'
+    # update PYTHONPATH so python can find mozregui package
+    if env.get('PYTHONPATH') != '':
```

Comment:
This line fails with an error for me:
ERROR: 'PYTHONPATH'

You should just write
```
if env.get('PYTHONPATH'):
```
This is because the get() method of a dict returns None by default when the key does not exist. and None != ''.
https://docs.python.org/2/library/stdtypes.html#dict.get